takwimu/settings.py

```diff
@@ -110,3 +113,48 @@
 ZENDESK_API = 'https://takwimu.zendesk.com/api/v2/requests.json'
 
 ZENDESK_API_TOKEN = os.environ.get('ZENDESK_API_TOKEN')
+
+# -------------------------------------------------------------------------------------
+# WAGTAIL Search
+# -------------------------------------------------------------------------------------
+
+
+TAKWIMU_ES_INDEX = os.environ.get('TAKWIMU_ES_INDEX', 'takwimu')
+TAKWIMU_ES_TIMEOUT = int(os.environ.get('TAKWIMU_ES_TIMEOUT', '5'))
```

Comment:
Is there a need to cast this to int? We could simply write as a number..

Also, 5 secs seems low - if the default on docker-compose is 30, should the default here also be 30?

cc @kilemensi @phillipahereza

Reply:
The use of 5 seconds and `int` comes directly from the documentation @DavidLemayian. But yes, the 5 seconds should definitely be changed to 30 here as well.